src/templates/document-definitions-shell-template.js

```diff
@@ -3,7 +3,7 @@ function(require) {
 
   var doc = { };
   var oldDoc = { };
-  var typeIdValidator = { };
+  var typeIdValidator = { type: 'string' };
```

Comment:
The `type` field is required for all property validators, even those that are simply stubs.